xc/xc7/tests/bufgmux/arty.sdc

```diff
@@ -0,0 +1,3 @@
+create_clock -period 10.0 clk_bufg -waveform {0.000 5.000}
+create_clock -period 8.0 clk0 -waveform {0.000 4.000}
```

Comment:
With the new clock propagation plug in, you should only have to constraint `clk`.

Reply:
fixed